runtime/bindings-generator.cs

```diff
@@ -1484,6 +1484,282 @@ static IEnumerable<FunctionData> GetFunctionData ()
 				}
 			);
 
+			data.Add (
+				new FunctionData {
+					Comment = " // IntPtr func (IntPtr, Vector2d, Vector2d, Vector2i, bool)",
+					Prefix = "simd__",
+					Variants = Variants.All,
```

Comment:
This can probably be `Variants.NonStret`, since the return value from the function isn't a structure.